sonar-scanner-engine/src/main/java/org/sonar/scanner/phases/AbstractPhaseExecutor.java

```diff
@@ -86,6 +96,25 @@ public final void execute(DefaultInputModule module) {
     eventBus.fireEvent(new ProjectAnalysisEvent(module, false));
   }
 
+  private void initCoverageExclusions() {
+    if (coverageExclusions.shouldExecute()) {
+      String stepName = "Init coverage exclusions";
+      eventBus.fireEvent(new BatchStepEvent(stepName, true));
+      coverageExclusions.log();
+
+      for (InputFile inputFile : fs.inputFiles(fs.predicates().all())) {
+        boolean excluded = coverageExclusions.isExcluded(inputFile);
+        if (excluded) {
+          ((DefaultInputFile) inputFile).setExcludedForCoverage(true);
+          LOG.debug("File {} excluded for coverage", inputFile.relativePath());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "relativePath"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)